docs/windows-design.md

```diff
@@ -216,18 +217,12 @@ It is processed and forwarded by OVS, and controlled with OpenFlow entries.
 
 ### Service Traffic
 
-Kube-proxy userspace mode is configured to provide Service functions. A specific Network Adapter named
-"HNS Internal NIC" is provided to kube-proxy to configure Service addresses. The OpenFlow entries for the Service
-traffic on Windows are the same as those on Linux.
+Kube-proxy userspace mode is configured to provide NodePort Service function. A specific Network Adapter named
+"HNS Internal NIC" is provided to kube-proxy to configure Service addresses. The OpenFlow entries for the
+NodePort Service traffic on Windows are the same as those on Linux.
 
-The Pod-to-Service case is similar to Antrea on Linux. A packet from a Pod destined to a Service leaves the OVS pipeline
-on the gateway interface first. kube-proxy captures the packet on the host and then chooses a valid endpoint. 
-The destination IP of the packet is changed to the endpoint, and enters the OVS pipeline again.
-
-One difference is that on Linux, the packet is using the client Pod's IP as source address when it re-enters the
-OVS pipeline, while on Windows, it uses the gateway IP as source. This is because userspace kube-proxy on Windows
-terminates the initial connection and sets up a new connection on the gateway interface to the chosen backend Pod, 
-after which the packet is forwarded back to OVS.
+The ClusterIP Service function is implemented by Antrea leveraging OVS. Antrea installs OpenFlow flows on OVS
```

Comment:
Antrea installs OpenFlow entries to select the backend endpoint and take DNAT on the Pod-to-CluterIP-Service traffic.

Reply:
Done.